doc/sphinx/credits.rst

```diff
@@ -817,12 +817,12 @@ expressions occurring in the goal by pattern in tactics such as set or
 destruct. Hugo Herbelin also relied on ideas from Chung-Kil Hur’s Heq
 plugin to introduce automatic computation of occurrences to generalize
 when using destruct and induction on types with indices. Stéphane Glondu
-introduced new tactics constr\_eq, is\_evar and has\_evar to be used
+introduced new tactics :tacn:`constr_eq`, :tacn:`is_evar`, and :tacn:`has_evar`, to be used
 when writing complex tactics. Enrico Tassi added support to fine-tuning
-the behavior of simpl. Enrico Tassi added the ability to specify over
+the behavior of :tacn:`simpl`:. Enrico Tassi added the ability to specify over
 which variables of a section a lemma has to be exactly generalized.
 Pierre Letouzey added a tactic timeout and the interruptibility of
-vm\_compute. Bug fixes and miscellaneous improvements of the tactic
+:tacv:`vm_compute`:. Bug fixes and miscellaneous improvements of the tactic
```

Comment:
Use `tacn`, not `tacv`.

Reply:
The documentation says:

    For example, you could link to the tactic variant above using :tacv:`simpl_at`

I naively thought the documentation was right.

Reply:
@cpitclaudel The documentation is wrong, right?

Reply:
Indeed, the documentation is wrong :disappointed: `tacv` and `warn` are the only exceptions to the rule that `.. xxx::` can be linked to using `:xxx:`.

https://github.com/coq/coq/blob/2a93216a3851688dd29c06a29c6d1442898faab8/doc/tools/coqrst/coqdomain.py#L1023-L1037

Reply:
This should be documented in README.rst. But I wonder whether we should fix the users or the code. Maybe `:tacv:` should be a synonym of `:tacn:`? The logic behind collapsing both `tacv` and `tacn` under the same role `tacn` was that it was a pain to have to distinguish when writing links. But it can also be confusing that `tacv` doesn't work :/

Reply:
Both solutions are OK to me. However, I'm quite surprised by the collapsing of `exn` and `warn`. These are not the same to me.